src/main/webapp/play.html

```diff
@@ -181,6 +221,47 @@
 
 	}
 	
+	function initializeVoDPlayer(name, token, noStreamCallback){
```

Comment:
I think this method should be moved to fetch.stream.js file.
Secondly, no need to have initilalizeHLsPlayer and initializeVoDPlayer separately. Just let us have one initializePlayer method